From 854df7cf5a1d4a69f348fe3cc58cf905693aada7 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Mon, 14 Apr 2025 13:49:51 +0200
Subject: [PATCH] man: mention that rhel-9.0 net naming scheme is the same as
 v251

Due to the rebase in 9.2 it is a bit confusing that we are documenting
the newer upstream scheme and it might look 9.0 is based in v255.
So let's explicitly mention that the scheme is the same as in v251.

Resolves: RHEL-87179

rhel-only: doc
---
 man/systemd.net-naming-scheme.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/man/systemd.net-naming-scheme.xml b/man/systemd.net-naming-scheme.xml
index 0b0bbbb2f1..d2dc5fe7bb 100644
--- a/man/systemd.net-naming-scheme.xml
+++ b/man/systemd.net-naming-scheme.xml
@@ -568,7 +568,8 @@
         <varlistentry>
           <term><constant>rhel-9.0</constant></term>
 
-          <listitem><para>Since version <constant>v247</constant> we no longer set
+          <listitem><para>Same as naming scheme <constant>v251</constant>.</para>
+          <para>Since version <constant>v247</constant> we no longer set
           <varname>ID_NET_NAME_SLOT</varname> if we detect that a PCI device associated with a slot is a PCI
           bridge as that would create naming conflict when there are more child devices on that bridge. Now,
           this is relaxed and we will use slot information to generate the name based on it but only if
